server/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -95,6 +95,18 @@ public void testPgConstraintTable() {
             "NULL| false| false| NULL| a| NULL| NULL| s| 0| a| 0| 0| true| NULL| t1_pk| -2048275947| true| NULL| NULL| 728874843| NULL| p| 0| true| -874078436\n"));
     }
 
+    @Test
+    public void testPgTablesTable() {
+        execute("select * from pg_tables WHERE tablename = 't1'");
+        assertThat(printedTable(response.rows()), is("false| false| false| false| doc| t1| NULL| NULL\n"));
+    }
+
+    @Test
+    public void testPgViewsTable() {
+        execute("select * from pg_views WHERE viewname = 'v1'");
+        assertThat(printedTable(response.rows()), is("SELECT \"id\"\nFROM \"doc\".\"t1\"\n| doc| v1| crate\n"));
```

Comment:
same here (check that table is not present)
```
execute("select * from pg_views WHERE viewname = 't1'");
```

Reply:
added as suggested 👍